build-live.sh

```diff
@@ -0,0 +1,304 @@
+lxc exec $CONTAINER -- apt install --no-install-recommends -y \
+  dnsmasq \
+  dnsmasq-utils \
+  hostapd \
+  iw \
+  rfkill \
+  rng-tools
+
+# grab the list of packages
+lxc exec $CONTAINER -- dpkg-query -W --showformat='${Package} ${Version}\n' | sort > ${CD}/${FS_DIR}/filesystem-desktop.manifest
+
+lxc exec $CONTAINER -- apt install -y --no-install-recommends casper
+
+echo "Diverting running-in-container..."
+lxc exec $CONTAINER -- dpkg-divert --local --rename --add /bin/running-in-container
+echo "Stubbing /bin/running-in-container..."
+echo | lxc file push --uid=0 --gid=0 --mode=755 - ${CONTAINER}/bin/running-in-container <<EOF
+#!/bin/sh
+[ -f /run/container_type ] && cat /run/container_type
+exit 0
+EOF
+
+echo "Creating policy-rc.d..."
+echo | lxc file push --uid=0 --gid=0 --mode 755 - ${CONTAINER}/usr/sbin/policy-rc.d <<EOF
+#!/bin/sh
+exit 101
+EOF
+
+# Move initctl out of the way, install dummy initctl
+echo "Diverting initctl..."
+lxc exec $CONTAINER -- dpkg-divert --local --rename --add /sbin/initctl
+echo "Linking..."
+lxc exec $CONTAINER -- ln -sf /bin/true /sbin/initctl
+
+# lxc exec $CONTAINER -- apt install -y ubuntu-desktop^ ubuntu-live^
+# lxc exec $CONTAINER -- apt install -y --no-install-recommends \
+lxc exec $CONTAINER -- apt install -y \
+  apt-clone \
+  archdetect-deb \
+  binutils \
+  crda \
+  cryptsetup \
+  cryptsetup-bin \
+  dmraid \
+  dpkg-repack \
+  ecryptfs-utils \
+  gir1.2-appindicator3-0.1 \
+  gir1.2-json-1.0 \
+  gir1.2-timezonemap-1.0 \
+  gir1.2-xkl-1.0 \
+  gparted \
+  iw \
+  keyutils \
+  kpartx \
+  kpartx-boot \
+  language-pack-en \
+  language-pack-en-base \
+  libatkmm-1.6-1 \
+  libcairomm-1.0-1 \
+  libcryptsetup4 \
+  libdebian-installer4 \
+  libdevmapper-event1.02.1 \
+  libdmraid1.0.0.rc16 \
+  libecryptfs0 \
+  libglibmm-2.4-1c2a \
+  libgtkmm-2.4-1c2a \
+  libnss3-1d \
+  libpangomm-1.4-1 \
+  libsigc++-2.0-0c2a \
+  libtimezonemap1 \
+  lvm2 \
+  pptp-linux \
+  python3-cairo \
+  python3-gi-cairo \
+  python3-icu \
+  python3-pam \
+  rdate \
+  sbsigntool \
+  ubiquity \
+  ubiquity-casper \
+  ubiquity-frontend-gtk \
+  ubiquity-ubuntu-artwork \
+  watershed \
+  wireless-regdb \
+  xserver-xorg \
+  gnome-themes-standard \
+  lightdm
+
+lxc exec $CONTAINER -- rm /usr/sbin/policy-rc.d
+
+echo "Removing diversions..."
+lxc exec $CONTAINER -- cp /bin/running-in-container.distrib /bin/running-in-container
+lxc exec $CONTAINER -- cp /sbin/initctl.distrib /sbin/initctl
+
+lxc exec $CONTAINER -- dpkg-divert --remove /bin/running-in-container
+lxc exec $CONTAINER -- dpkg-divert --remove /sbin/initctl
+
+# clean up
+
+lxc exec $CONTAINER -- apt-get clean
+lxc exec $CONTAINER -- rm /var/lib/dbus/machine-id
+lxc exec $CONTAINER -- rm /etc/init/console.override
+lxc exec $CONTAINER -- rm /etc/init/tty1.override
+lxc exec $CONTAINER -- rm /etc/init/tty2.override
+lxc exec $CONTAINER -- rm /etc/init/tty3.override
+lxc exec $CONTAINER -- rm /etc/init/tty4.override
```

Comment:
I have no idea where these overrides come in (they're present in the LXD `ubuntu:14.04` image). With them present, it's not possible to log into a device using a terminal.

`cloud-init` may be related, as its presence (I think) may have been responsible for keyboard input not being accepted on the TTYs at first (so you could see a login prompt via HDMI but not actually log in).